doc/source/examples/permutation_importance_classification_leave.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>"which in our case since it means that they are not relevant for the model" - rewrite, e.g. "which means that they are not relevant for the model"</p><p> </p><p>can simplify last sentence, e.g. "It is worth emphasizing that this does not always happen - using a different metric or loss function can give different results".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>